patches/bnc-onboard++web3-provider-engine+15.0.12.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/node_modules/bnc-onboard/node_modules/web3-provider-engine/index.js b/node_modules/bnc-onboard/node_modules/web3-provider-engine/index.js
+index 3a1b7d5..9bf0f83 100644
+--- a/node_modules/bnc-onboard/node_modules/web3-provider-engine/index.js
++++ b/node_modules/bnc-onboard/node_modules/web3-provider-engine/index.js
+@@ -114,7 +114,11 @@ Web3ProviderEngine.prototype.send = function(payload){
+   throw new Error('Web3ProviderEngine does not support synchronous requests.')
+ }
+ 
+-Web3ProviderEngine.prototype.sendAsync = function(payload, cb){
++Web3ProviderEngine.prototype.sendAsync = function (payload, cb) {
```

Comment:
If you restore the spaces how they were, this line would be unaffected.
Looks very clean otherwise!